namespaces/live.cloud-platform.service.justice.gov.uk/money-to-prisoners-test/resources/irsa.tf

```diff
@@ -1,17 +1,25 @@
 # generated by https://github.com/ministryofjustice/money-to-prisoners-deploy
 module "irsa-api" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=1.1.0"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=2.0.0"
 
-  namespace        = var.namespace
-  eks_cluster_name = var.eks_cluster_name
-  role_policy_arns = [aws_iam_policy.analytical-platform.arn]
-  service_account  = "${var.namespace}-api"
-  # NB: service account name must be unique within Cloud Platform (IAM role name is derived from it)
+  business_unit          = var.business_unit
+  team_name              = var.team_name
+  application            = var.application
+  is_production          = var.is_production
+  namespace              = var.namespace
+  environment_name       = var.environment
+  infrastructure_support = var.infrastructure_support
+
+  eks_cluster_name     = var.eks_cluster_name
+  service_account_name = "api"
```

Comment:
i am aware that this will re-create the service account under a new name, it's deliberate